src/Azure.Functions.Cli/Azure.Functions.Cli.csproj

```diff
@@ -154,7 +154,7 @@
     <PackageReference Include="YamlDotNet" Version="6.0.0" />
   </ItemGroup>
   <ItemGroup Condition="'$(NoWorkers)' != 'true'">
-    <PackageReference Include="Microsoft.Azure.Functions.JavaWorker" Version="1.8.2-SNAPSHOT" />
+    <PackageReference Include="Microsoft.Azure.Functions.JavaWorker" Version="1.8.2-test" />
```

Comment:
Why do we use 1.8.2-test?

Reply:
well, that was an accident.... i'll revert. (trying to write code while in another meeting :-))